CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.0.0]
+### Changed
+- **BREAKING**: Split AssetsDetectionController into CollectiblesDetectionController and TokenDetectionController ([#619](https://github.com/MetaMask/controllers/pull/619))
+  - Consumers of the AssetsDetectionController will have to now import both TokenDetectionController and CollectibleDetectionController and split up the calling of any methods accordingly.
+- **BREAKING**: Set the `conversionDate` to `null` if fetching the data fails. (#621)
```

Comment:
Sorry for the back and forth :sweat_smile:  One more suggestion:

```suggestion
- **BREAKING**: Set the `CurrencyRateController` property `conversionDate` to `null` if fetching the data fails. (#621)
  - Consumers of the `CurrencyRateController` will need to ensure their code anticipates that `conversionDate` will sometimes be set to `null`.
```

I noticed after re-reading that I wasn't following our guideline of offering migration instructions for each breaking change.

Reply:
Don't worry, thanks for the feedback!